k8s/namespaces/et/et-cos/prod.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
You shouldn't be needing anything i thought on k8s directory  as everything seems to be  flux v2 ?

Reply:
@adusumillipraveen - The repo ReadMe states "Prod | Flux V1 & V2 | Migration in progress". We therefore assumed both V1 & V2 changes are included. We can remove V1 code from the PR if you can update the ReadMe?

Reply:
I think the status is to reflect that some apps still use v1 as migration is in progress, but new teams always go to v2. One app cannot be maintained on both. @hannah38 pls correct me if wrong.

Reply:
Hi yes, that's right. It says migration in progress to indicate that some teams are still to be migrated - but new teams should be flux v2 only.

Reply:
@adusumillipraveen @hannah38 - Okay no problem I'll remove the Flux V1 code from the PR and let you know once done.

Reply:
@adusumillipraveen - Okay Praveen, now removed Flux V1 shell script changes from the PR. Please can you review again, thanks.

Reply:
@petermoores-version1 - in another PR , could you cleanup/move the other envs to flux v2 as well?